include/libdnf5/conf/vars.hpp

```diff
@@ -86,7 +95,13 @@ struct Vars {
     /// @param name Name of the variable
     const Variable & get(const std::string & name) const { return variables.at(name); }
 
-    static std::unique_ptr<std::string> detect_release(const BaseWeakPtr & base, const std::string & install_root_path);
+    static std::optional<std::string> detect_release(const BaseWeakPtr & base, const std::string & install_root_path);
```

Comment:
I am sorry, but we cannot use std::optional because it is not supported by SWIG.

Reply:
Ah, forgot about that, that's a shame. I'll use `unique_ptr` instead.